src/main/java/com/google/ehub/data/CommentDataManager.java

```diff
@@ -0,0 +1,43 @@
+package com.google.ehub.data;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+import java.util.*;
+
+public class CommentDataManager{
+
+  public void addItemComment(long itemId, String message, long timestamp){
+    Entity commentEntity = new Entity("ItemPageComments");
+    commentEntity.setProperty("itemId", itemId);
+    commentEntity.setProperty("message", message);
+    commentEntity.setProperty("timestamp",timestamp);
+
+    DatastoreService commentDatastoreInitialization = DatastoreServiceFactory.getDatastoreService();
```

Comment:
Nit: `commentDatastoreInitialization` can be renamed to `datastore`.

Reply:
Reminder here. Its good to have descriptive variable names but `commentDatastoreInitialization` seems a little too excessive.